### PR TITLE
Opentracing 1.1 initial code

### PR DESCRIPTION
**Pull request by fmhwong**
- Upgraded io.opentracing APIs from 0.30 to 0.31
- Upgraded org.eclipse.microprofile.opentracing API from 1.0 to 1.1
- Removed `ActiveSpan`

### CODE REVIEW

File: dev/com.ibm.ws.opentracing.1.1/resources/com/ibm/ws/opentracing/resources/Opentracing.nlsprops [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+OPENTRACING_NO_APPNAME_FOUND_IN_JNDI.useraction=Determine why the application name is not available through JNDI lookup of "java:app/AppName".
+
+OPENTRACING_COULD_NOT_CREATE_TRACER=CWMOT0001E: Invocation of user supplied OpentracingTracerFactory.newInstance(...) method failed with Exception. Message = {0}
+OPENTRACING_COULD_NOT_CREATE_TRACER.explanation=The feature constructs Tracer implementations with a user provided OpentracingTracerFactory. The OpentracingTracerFactory.newInstance method returned an exception.
+OPENTRACING_COULD_NOT_CREATE_TRACER.useraction=Determine why the user supplied OpentracingTracerFactory newInstance method returned an exception.
+
+OPENTRACING_TRACERFACTORY_RETURNED_NULL=CWMOT0006E: Invocation of user supplied OpentracingTracerFactory.newInstance(...) method returned null
+OPENTRACING_TRACERFACTORY_RETURNED_NULL.explanation=The feature constructs Tracer implementations with a user provided OpentracingTracerFactory. The OpentracingTracerFactory.newInstance method returned null.
+OPENTRACING_TRACERFACTORY_RETURNED_NULL.useraction=Determine why the user supplied OpentracingTracerFactory newInstance method returned null.
```
Comment (lzdanski) [inline dev/com.ibm.ws.opentracing.1.1/resources/com/ibm/ws/opentracing/resources/Opentracing.nlsprops]: OPENTRACING_TRACERFACTORY_RETURNED_NULL=CWMOT0006E: Invocation of **a** user supplied OpentracingTracerFactory.newInstance(...) method returned null

OPENTRACING_TRACERFACTORY_RETURNED_NULL.explanation=The feature constructs Tracer implementations with a user provided **value for** OpentracingTracerFactory. *The OpentracingTracerFactory.newInstance method returned null.* **Should we include info about what the method should return?**

OPENTRACING_TRACERFACTORY_RETURNED_NULL.useraction=Determine why the user supplied OpentracingTracerFactory newInstance method returned null. **How does the developer accomplish this? Is it by examining their own work or by reaching out to the user? Are there any resources to direct them to?**

File: dev/com.ibm.ws.opentracing.1.1/resources/com/ibm/ws/opentracing/resources/Opentracing.nlsprops [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+OPENTRACING_NO_APPNAME_FOUND_IN_JNDI=CWMOT0000E: Lookup of the application name in JNDI did not return a value. A default name is used for Opentracing service name.
+OPENTRACING_NO_APPNAME_FOUND_IN_JNDI.explanation=The feature provides the application name to the user supplied OpentracingTracerFactory implementation. The application name could not be retrieved by JNDI lookup of "java:app/AppName". A default service name was provided.
+OPENTRACING_NO_APPNAME_FOUND_IN_JNDI.useraction=Determine why the application name is not available through JNDI lookup of "java:app/AppName".
+
+OPENTRACING_COULD_NOT_CREATE_TRACER=CWMOT0001E: Invocation of user supplied OpentracingTracerFactory.newInstance(...) method failed with Exception. Message = {0}
+OPENTRACING_COULD_NOT_CREATE_TRACER.explanation=The feature constructs Tracer implementations with a user provided OpentracingTracerFactory. The OpentracingTracerFactory.newInstance method returned an exception.
+OPENTRACING_COULD_NOT_CREATE_TRACER.useraction=Determine why the user supplied OpentracingTracerFactory newInstance method returned an exception.
```
Comment (lzdanski) [inline dev/com.ibm.ws.opentracing.1.1/resources/com/ibm/ws/opentracing/resources/Opentracing.nlsprops]: OPENTRACING_COULD_NOT_CREATE_TRACER=CWMOT0001E: Invocation of **the** user supplied OpentracingTracerFactory.newInstance(...) method failed with **an exception**. Message = {0}

OPENTRACING_COULD_NOT_CREATE_TRACER.explanation=The feature constructs Tracer implementations with a user provided OpentracingTracerFactory. The OpentracingTracerFactory.newInstance method returned an exception.

OPENTRACING_COULD_NOT_CREATE_TRACER.useraction=Determine why the user supplied OpentracingTracerFactory newInstance method returned an exception. **How does the developer accomplish this? Is it by examining their own work or by reaching out to the user? Are there any resources to direct them to?**

File: dev/com.ibm.ws.opentracing.1.1/resources/com/ibm/ws/opentracing/resources/Opentracing.nlsprops [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+# For more information, visit: 
+# http://washome.austin.ibm.com/xwiki/bin/view/MessagesTeam/
+#
+
+#-----------------------------------------------------------------------------------------------------------------------------
+# Messages used by Opentracing Impl -  range is 0000-4999, plus 9999 for the emergency message
+#-----------------------------------------------------------------------------------------------------------------------------
+OPENTRACING_NO_APPNAME_FOUND_IN_JNDI=CWMOT0000E: Lookup of the application name in JNDI did not return a value. A default name is used for Opentracing service name.
+OPENTRACING_NO_APPNAME_FOUND_IN_JNDI.explanation=The feature provides the application name to the user supplied OpentracingTracerFactory implementation. The application name could not be retrieved by JNDI lookup of "java:app/AppName". A default service name was provided.
+OPENTRACING_NO_APPNAME_FOUND_IN_JNDI.useraction=Determine why the application name is not available through JNDI lookup of "java:app/AppName".
```
Comment (lzdanski) [inline dev/com.ibm.ws.opentracing.1.1/resources/com/ibm/ws/opentracing/resources/Opentracing.nlsprops]: OPENTRACING_NO_APPNAME_FOUND_IN_JNDI.useraction=Determine why the application name is not available **by using** JNDI lookup of "java:app/AppName".

File: dev/com.ibm.websphere.appserver.features/visibility/public/opentracing-1.1/resources/l10n/com.ibm.websphere.appserver.opentracing-1.1.properties [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+################################################################################
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature provides support for Opentracing function for enterprise Java.
```
Comment (lzdanski) [inline dev/com.ibm.websphere.appserver.features/visibility/public/opentracing-1.1/resources/l10n/com.ibm.websphere.appserver.opentracing-1.1.properties]: description=This feature supports the Opentracing function for **Enterprise Java.**

Does this need to be more specific? Do we need to include the name of the feature?

**Comment by fmhwong** [comment]
@lzdanski I forgot to mention that these are straight copy from mpOpentracing-1.0.  That means that have been shipped.  I didn't write these messages and we recently picked up the development responsibility.  Do you still want to make these changes, we have to make it in both mpOpentracing-1.0 and 1.1.  Please let me know your thoughts.

**Comment by fmhwong** [comment]
#build

**Comment by lzdanski** [comment]
@fmhwong Since these are shipped, disregard my comments and I'll go ahead and add the "ID review not needed" label.

**Comment by fmhwong** [comment]
#build

**Comment by fmhwong** [comment]
Last good personal build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_fj4z4JfpEeiA46HHn3eUcA

**Pull request merged**